.env.test

```diff
@@ -1,23 +1,16 @@
-# direnv will make a copy of this file as an .env file
-# Follow the instructions in README.md as how to add your personal
-# tokens and secrets in this file
-
-# GitHub App ID
+# GitHub
+FORCE_USER_TOKEN_GITHUB_CLIENT="false"
+GH_USER_TOKEN="ghp_BLAHBLAHBLAH"
 GH_APP_IDENTIFIER="1234"
-# Github App Webhook Secret to verify requests come from GitHub
 GH_WEBHOOK_SECRET="webhooksecret"
-# GitHub App Secret Key
-# NOTE: this *must* be on a single line, otherwise it will break
-# reading the secret key.
-GH_APP_SECRET_KEY='-----BEGIN RSA PRIVATE KEY----------END RSA PRIVATE KEY-----'
+GH_APP_SECRET_KEY="top \nsecret\n key"
```

Comment:
Switching to a more testable value, also exercising the `\n`-stripping behavior.